### PR TITLE
Implemented state_string function

### PR DESCRIPTION
**Pull request by kjav**

### CODE REVIEW

**Comment by kjav** [comment]
This is the last function in the Connection class that is not yet implemented. There are still functions in the Context class which are not implemented, but do not throw NotImplemented exceptions (e.g. load_client_ca)

**Comment by alex** [comment]
Looks ok to me, @hynek can you confirm?

**Comment by hynek** [comment]
Needs a changelog entry, otherwise seems ok.

**Comment by kjav** [comment]
Is the format of that entry ok?

File: ChangeLog [hunk 1]
```diff
@@ -1,3 +1,9 @@
+2015-09-07 Sam Lee <sam.rr.lee@gmail.com>
+
+	* OpenSSL/SSL.py, OpenSSL/test/test_ssl.py: Implemented
+	  the empty ``Connection.state_string()`` function, using
```
Comment (alex) [inline ChangeLog]: Can you remove the word "empty", I think it's confusing unless you know the full history of this bug.

**Comment by kjav** [comment]
OK, that makes sense. I have removed it.

**Comment by hynek** [comment]
thanks!

**Pull request merged**